src/app/modules/course-template/components/course-template/course-template.component.html

```diff
@@ -16,6 +18,17 @@
       title="{{lesson.fields.lessonTitle}}" description="{{content.convertRichText(lesson.fields.lessonDescription)}}">
     </app-course-bar-item>
   </div>
+
+  <div class="nextButtons">
+    <h2 *ngIf="!nextCourse">Congrats You Have Finished All the Courses!</h2>
```

Comment:
I think we should only have a back button and hide the next button if there are no more courses/lessons. We can do this in a separate PR as part of our sanitization for each page